Assets/Scripts/Lua/CLRBindings/LuaArenaStatus.cs

```diff
@@ -79,4 +79,19 @@ public class LuaArenaStatus {
         } else
             UnitaleUtil.DisplayLuaError("NOT THE WAVE SCRIPT", "sorry but pls don't");
     }
+
+    public void SetSprite(string filename) {
```

Comment:
You added Set functions but no Get functions; for consistency you might just want to make these variables (like `Arena.color32 = {}`)

Reply:
Also honestly, you might want to see if you can just find a way to get a sprite object from the Arena and be able to modify it through `Arena.inner_sprite` and `Arena.outer_sprite` or something.

Reply:
I wanted to give the full access to the sprite (and also access outside wave scripts), but that would give an opportunity to break the UI, and I guess that's not what we want?

Reply:
Changing how something appears won't change functionality, there's no reason for something like this to be restricted to the wave script. What's the point of letting the modder modify these if it can only be done in waves?